stdlib/event/time.lua

```diff
@@ -36,44 +39,54 @@ Event.Time.daily = script.generate_event_name()
 Event.register(defines.events.on_tick,
     function()
         for idx, surface in pairs(game.surfaces) do
-            local day_time = math.fmod(Event.Time.get_day_time(idx), 1)
-            local day_time_minutes = math.floor(day_time * 24 * 60)
+            if not surface.freeze_daytime then
+                local day_time = math.fmod(get_day_time(idx), 1)
+                local day_time_minutes = math.floor(day_time * 24 * 60)
```

Comment:
I will make the change

Re: indenting  I added an addition if block so everything inside of it got indented  (additionally I probably had format on save turned on)

The one thing I don't like about the lua "best practices" is their suggestion to do
`if fail then return end`

I tend to do logical ordered statements
```
if good then
  ...
else -- not good
  return
end
```